src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -812,11 +818,11 @@ deferredInstallPrompt.prompt();
 evt.srcElement.setAttribute('hidden', true);
 ```
 
-Al llamar a `.prompt()` se mostrará un diálogo modal al usuario y se le pedirá que agregue su aplicación a la pantalla de inicio.
+Al llamar a `.prompt()` se mostrará un diálogo modal al usuario y se le pedirá que agrega tu aplicación a la pantalla de inicio.
```

Comment:
se le pedirá que **agregue** tu aplicación

Reply:
I changed by `se te pedirá que agregues` to keep the conjugation in the second person.